docs/platform/howto/create_new_service.rst

```diff
@@ -0,0 +1,48 @@
+﻿Creating a new Aiven service
+============================
+
+You can add services for the various Aiven products in the Aiven web console.
+
+To create a new service:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
+
+2. On the *Services* page, click **Create a new service**.
+
+   This opens a new page with the available service options.
+
+   .. image:: /images/tools/console/console_create_service.png
+
+3. Select the main properties for your service:
+
+   a. Select the service type.
```

Comment:
These seem to render as numbers, rather than letters, is that a problem?

Reply:
I guess that's something that could be solved in the CSS - it seems to be the case for all nested ol elements, but I don't think it's a huge issue ;)